src/scss/components/_modal.scss

```diff
@@ -0,0 +1,35 @@
+.bp-modal {
+  position: fixed;
+  top: 0; left: 0;
+  width: 100%; height: 100%;
+  z-index: 1000;
+
+  &__overlay {
+    width: 100%; height: 100%;
+    top: 0; left: 0;
+    position: absolute;
+    background: rgba(0, 0, 0, 0.7);
+    z-index: 1;
+  }
+
+  &__container {
+    position: relative;
+    z-index: 2;
+    background: #fff;
+    margin: 12*$m auto 0 auto;
```

Comment:
Is a fixed margin a good thing in this case? The container must be in the middle of the screen, no?

Reply:
Good point!